### DISCUSSION TITLE
Tasmota documentation example of SWB1 (Smart Power Strip) flashing appears to have several errors?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [arendst/Tasmota#17681] [Q&A]
Documentation that appears to have a broken link to a now missing WT8266 datasheet?
https://tasmota.github.io/docs/devices/SWB1-Smart-Power-Strip/

The broken link is on the page segment label JTAG at the top.

I found a live link to the datasheet at...
[https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&ved=2ahUKEwjZ6r7Jjb78AhW8J0QIHVrZBz0QFnoECBEQAQ&url=https%3A%2F%2Fwww.seeedstudio.com%2Fdocument%2Fword%2FWT8266-S1%2520DataSheet%2520V1.0.pdf&usg=AOvVaw3tw6Y9Nxn5jg-Zed2x509c
](url)

Last, the picture of the IC pin assignment shows an unpopulated 5 pin header to the lower right of the WT8266 IC.  Can someone update the documentation to show the pin assignment on this header?  I tried to do continuity checks on the traces to this header, but I don't think my cheap multi-meter is working right, otherwise I would suggest the pin assignment here.

**Comment by barbudor** [comment]
Dear @Jibun-no-Kage 

Documentation is done by voluntary contributors so it's up to anyone here around, including YOU to help maintaining the documentation. This can be done by clicking on the pencil icon on the top right of the page.

Indeed the web site address is wrong (there is no ending-'s' wirelass-tag.com)
The exact manufacturer datasheet v2.5 is present on his [web site](http://www.wireless-tag.com/portfolio/wt8266-s1-2/) and confirm the exact pinout that we have documented (just like the old v1.0 you are pointing).

Regarding the undocumented connector on the right, you have the device in hand so you can probably figure out where pins are connected.

Thank you in advance for your forthcoming contribution to Tasmota documentation.

**Comment by Jibun-no-Kage** [comment]
Thanks for the clarification and information.
